src/python/tests/core/bot/fuzzers/syzkaller/Data/crash

```diff
@@ -0,0 +1,19 @@
+c5  11608 SELinux: unrecognized netlink message: protocol=0 nlmsg_type=5120 sclass=netlink_route_socket
```

Comment:
Rename Data to test_data.
Rename crash to crash.txt

Reply:
Done